wiki/Beatmap_search/de.md

```diff
@@ -0,0 +1,118 @@
+---
+tags:
+  - find beatmaps
+  - filter beatmaps
+  - query beatmaps
+  - client search
+  - website search
+  - suche Beatmaps
+---
+
+# Beatmap-Suche
+
+*Siehe auch: [Beatmap § Beatmaps herunterladen](/wiki/Beatmap#beatmaps-herunterladen)*
+
+Standardmäßig wird jeder Text, der in der [Songauswahl](/wiki/Client/Interface#songauswahl) oder im Suchfeld auf der Webseite eingeben wird, mit dem Künstler, dem Titel, dem Ersteller, dem Namen, der Quelle und den Tags der Beatmap abgeglichen. Die Angabe einer einzelnen Zahl zeigt Beatmaps und Schwierigkeitsgrade mit einer bestimmten [Kennung](/wiki/Beatmap#identifizierung) an. Bei allen Suchanfragen wird die Groß- und Kleinschreibung nicht berücksichtigt.
+
+Metadaten von Beatmaps können für eine genauere Filterung über die folgenden Operatoren mit bestimmten Werten verglichen werden:
+
+| Vergleich | Beschreibung |
+| :-: | :-- |
+| `=` oder `==` | Gleich |
+| `!=` | Nicht gleich |
+| `<` | Kleiner als |
+| `>` | Größer als |
+| `<=` | Kleiner als oder gleich |
+| `>=` | Größer als oder gleich |
+
+## Client
+
+*Anmerkung: Diese Filter funktionieren nur in der Songauswahl, während osu!direct normale Volltextsuche unterstützt.*
+
+| Filter | Beschreibung |
+| :-: | :-- |
+| `artist` | Name des Künstlers |
+| `creator` | Name des Beatmap-Erstellers |
+| `title` | Songtitel |
+| `difficulty` | Name des Schwierigkeitsgrads der Beatmap |
+| `ar` | [Approach-Rate](/wiki/Beatmap/Approach_rate) |
+| `cs` | [Circle-Size](/wiki/Beatmap/Circle_size) |
+| `od` | [Allgemeine Schwierigkeit](/wiki/Beatmap/Overall_difficulty) |
+| `hp` | [HP Drain-Rate](/wiki/Beatmap/HP_drain_rate) |
+| `key`, `keys` | Anzahl der Keys (betrifft nur osu!mania und konvertierte Beatmaps) |
+| `star`, `stars` | [Sternebewertung](/wiki/Beatmap/Star_rating) |
+| `bpm` | [Songtempo](/wiki/Music_theory/Tempo) |
+| `length` | [Länge](/wiki/Beatmap/Play_time) in Sekunden |
```

Comment:
```suggestion
| `length` | [Dauer](/wiki/Beatmap/Play_time) in Sekunden |
```
Länge oder Dauer?

(Keep in mind this appears several times in this article)

Reply:
I think Länge is fine